### PR TITLE
Two bug fixes

### PR DESCRIPTION
**Pull request by chrise96**
- No extend was used to add values in a list in a for loop
- Added try except with Decos parse XML
- In postprocessing a reordering was done on a list of data that needed to be immutable.

### CODE REVIEW

File: postprocessing.py [hunk 1]
```diff
@@ -442,22 +446,15 @@ def calculate_score(bridge_distance: float, permit_distance: float) -> float:
...
         table_name = "containers"
 
         # Get images with a detection
         # TODO: perform sanitizing inputs SQL. For now, code will break if start_date_dag_ymd is not a datetime.
         sql = (
-            f"SELECT * FROM detections A LEFT JOIN images B ON A.file_name = B.file_name WHERE "
-            f"date_trunc('day', taken_at) = '{start_date_dag_ymd}'::date;"
+            f"SELECT B.file_name, B.heading, B.camera_location_lat, B.camera_location_lon FROM detections A "
+            f"LEFT JOIN images B ON A.file_name = B.file_name WHERE "
+            f"date_trunc('day', taken_at) = '2023-01-11'::date;"
```
Comment (chrise96) [inline postprocessing.py:457]: 2023-01-11

File: retrieve_images.py [hunk 2]
```diff
@@ -96,51 +96,51 @@ def get_pano_ids(start_date_dag_ymd: str) -> Any:
...
-        my_format_ymd = "%Y-%m-%d"
-        start_date = datetime.strptime(start_date_dag_ymd_item, my_format_ymd)
-        end_date = start_date + timedelta(days=1)
-        one_day_later = end_date.strftime(my_format_ymd)
-
-        pano_url = (
-            f"https://api.data.amsterdam.nl/panorama/panoramas/?srid=28992&timestamp_after={start_date_dag_ymd_item}"
-            f"&timestamp_before={one_day_later}"
-        )
+    start_date_dag_ymd = "2023-01-11"  # TODO remove!!!
```
Comment (chrise96) [inline retrieve_images.py:99]: 2023-01-11

File: visualizations/utils.py [hunk 3]
```diff
@@ -67,11 +72,14 @@ def is_container_permit(permit: Any) -> bool:
             "cabin",
         ]
         description = permit.find("TEXT8")
-        if any(
-            get_close_matches(word, container_words)
-            for word in description.text.split(" ")
-        ):
-            return True
+        try:
+            if any(
```
Comment (thomasbrockmeier-ams) [inline visualizations/utils.py:76]:
```
return any(get_close_matches(word, container_words) for word in description.text.split(" "))
```
is iets netter

File: retrieve_images.py (hunk 2 above)
Comment (thomasbrockmeier-ams) [reply retrieve_images.py:99]: moeten we dit nog steeds hard coden?

**Review by thomasbrockmeier-ams — APPROVED** [review]

**Pull request merged**